idseq_pipeline/commands/host_filtering_functions.py

```diff
@@ -357,8 +359,12 @@ def unmapped_files_in(some_dir):
             run_star_part(tmp_result_dir, genome_part, unmapped)
             unmapped = sync_pairs(unmapped_files_in(tmp_result_dir))
     else:
-        run_star_part(SCRATCH_DIR, genome_dir, fastq_files)
+        # if index was not partitioned, can make host gene-counts output file
+        run_star_part(SCRATCH_DIR, genome_dir, fastq_files, True)
         unmapped = sync_pairs(unmapped_files_in(SCRATCH_DIR))
+        gene_counts = os.path.join(SCRATCH_DIR, "ReadsPerGene.out.tab")
+        execute_command("aws s3 cp --quiet %s %s/;" % (gene_counts, SAMPLE_S3_OUTPUT_PATH))
```

Comment:
do you want to copy them to a slightly different file likst STAR.ReadsPerGene.out.tab